src/scheduler/fifo.c

```diff
@@ -2608,7 +2608,11 @@ update_svr_schedobj(int connector, int cmd, int alarm_time)
 		return 1;
 
 	/* Stat the scheduler to get details of sched */
-	ss = pbs_statsched(connector, sc_name, NULL, NULL);
+	for(ss = pbs_statsched(connector, NULL, NULL);
+			((ss != NULL) && strcmp(sc_name, ss->name));
+			ss = ss->next
+		);	//empty for loop
```

Comment:
Don't use C++ style comments.  Use /* */.  Also, since you are searching this multiple times, consider writing a function find_batch_status() and put it in libcmds.  It could be useful in the future.

Reply:
>Don't use C++ style comments. Use /* */.

The comment style is changed to /* */ in the second commit

>Also, since you are searching this multiple times, consider writing a function find_batch_status() and put it in libcmds. It could be useful in the future.

I understand it would be help full in future to add this new utility function,
However I like to pass on this for now, since I'm busy with other major RFE, and I'm repeating this code only twice for now.

But if you really insist, how about adding below macro in pbs_ifl.h or 
```
for_each_bach_status_sc( con, bs, sc_name )    for(\
                                             bs = pbs_statsched(con, NULL, NULL); \
                                             ((bs!= NULL) && strcmp(sc_name, bs->name));\
                                             bs = bs->next ) 
```